### ISSUE TITLE
CSS classes/styles too general

### ISSUE THREAD

**Issue by fritzmg** [FACT-Finder-Web-Components/magento2-module#79]
The CSS styles included with this extension should be changed be more specific (and may be the generated HTML structure of the FACT Finder Webcomponents need to be changed too).

For example, the magento2-module includes the following CSS: 
https://github.com/FACT-Finder-Web-Components/magento2-module/blob/050c51d870112124aac17c8a97ff0d7ea8133f98/Omikron/Factfinder/view/frontend/web/css/ff/suggest.css#L34-L40

This will affect _all_ product names across the whole shop - at least with the default Magento themes (blank and luma), not just the suggest element.

Another issue occurs with the `.ff-suggest` class. The module includes the following CSS:
https://github.com/FACT-Finder-Web-Components/magento2-module/blob/050c51d870112124aac17c8a97ff0d7ea8133f98/Omikron/Factfinder/view/frontend/web/css/ff/suggest.css#L1-L4

The FACT Finder Webcomponents generate the following HTML:
```html
<ff-suggest layout="block" class="x-scope ff-suggest-0">
    <div style="position: relative; width: 100%" class="style-scope ff-suggest">
        <div id="ffSuggestContainerWrapper" class="style-scope ff-suggest hideffSuggestContainerWrapper">
            <div id="ffSuggestContainer" class="style-scope ff-suggest blockLayout">
                <section id="searchContainer" class="searchTermContainer">
                    <!-- … -->
                </section>
                <section id="productContainer" class="productsContainer">
                    <!-- … -->
                </section>
            </div>
        </div>
    </div>
</ff-suggest>
```
So in total there are 3 HTML elements that have the CSS class `ff-suggest` and they are all nested. Thus the background color and box-shadow is applied to all of these.

**Comment by a-laurowski** [comment]
Hi @fritzmg 
Thanks for reporting. We will change the default module stylesheet to be more specific following Your suggestions. However the Web Components are not part of Magento 2 module thus I'm not be able to change its templates html as they are part of core of FACT-Finder Web Components which are platform independent

**Comment by fritzmg** [comment]
> However the Web Components are not part of Magento 2 module thus I'm not be able to change its templates html as they are part of core of FACT-Finder Web Components which are platform independent

Yes, I realize that, just wanted to mention it anyway.

**Comment by a-laurowski** [comment]
I'm closing this issue. CSS changes was merged in #81 and is a part of v0.9-beta.8
